release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -63,6 +63,13 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-10-networking"]
 === Networking
 
+[id="ocp-4-10-openshift-sdn-netpol-egress-policies"]
+==== OpenShift SDN cluster network provider network policy support for egress policies and ipBlock except
+
+If you use the OpenShift SDN cluster network provider, you can now use egress policy in your network policies. You define egress policies in the `egress` array of the `NetworkPolicy` object. You can also use the `except` clause when defining a network policy that uses the `ipBlock` field.
```

Comment:
Could this be rephrased to say "you can now use egress rule in network policy with IpBlock and ipBlock except".

Reply:
@asood-rh, updated.

Reply:
Thanks! @jboxman-rh